docs/blocks/block-controls-toolbars-and-inspector.md

```diff
@@ -159,10 +159,16 @@ registerBlockType( 'gutenberg-boilerplate-esnext/hello-world-step-04', {
 ```
 {% end %}
 
-Note that `BlockControls` is only visible when the block is currently selected.
+Note that `BlockControls` is only visible when the block is currently selected and in visual editing mode. `BlockControls` are not shown when editing a block in HTML editing mode.
 
 ## Inspector
 
-While the toolbar area is useful for displaying controls to toggle attributes of a block, sometimes you will find that you need more screen space for form fields. The inspector region is shown in place of the post settings sidebar when a block is selected.
+<img src="https://raw.githubusercontent.com/WordPress/gutenberg/try/4413-format-options-html/docs/blocks/inspector.png" with="281" height="527" alt="inspector">
```

Comment:
It will, but I wanted it in here until we merged so people could see it 😄

I plan to change it to master before merge, but you're right: I should've mentioned it 😄